pyspecdata/DCCT_function.py

```diff
@@ -183,22 +186,41 @@ def DCCT(
     )
     minorLocator = lambda: mticker.AutoMinorLocator(n=5)
     LHS_labels, _ = fig.transFigure.inverted().transform(
-        (label_spacing_multiplier * num_dims + allow_for_ticks_default, 0)
+        (vert_label_spacer * num_dims, 0)
     )
-    width = 1.0 - (LHS_pad + RHS_pad + LHS_labels)
+    width = bbox[2] - LHS_labels
     for j, b in enumerate(axes_bottom):
         if j != 0 and shareaxis:
             ax_list.append(
                 plt.axes(
-                    [LHS_labels + LHS_pad, b, width, axes_height],
+                    [LHS_labels + bbox[0], b, width, axes_height],
                     sharex=ax_list[0],
                     sharey=ax_list[0],
                 )
-            )  # lbwh
+            )  # in figure coords: x, y, width, height
         else:
             ax_list.append(
-                plt.axes([LHS_labels + LHS_pad, b, width, axes_height])
+                plt.axes([LHS_labels + bbox[0], b, width, axes_height])
             )  # lbwh
+    # {{{ make blended transform for plotting coherence transfer labels
+    dx = LHS_labels + bbox[0]  # x coord in figure coord
+    axis_to_figure = ax_list[0].transAxes + fig.transFigure.inverted()
+    _, dy = axis_to_figure.transform(
+        r_[0, 0]
+    )  # bottom left corner of first ax in fig coord
+    total_scale_transform = IdentityTransform() + ScaledTranslation(
+        dx, dy, fig.transFigure
+    )
+    total_trans = blended_transform_factory(
+        total_scale_transform, fig.transFigure
+    )
+    # }}}
+    # {{{ transform for arrows
+    ax_x, ax_y = axis_to_figure.transform(r_[0, 0])
+    arrow_trans = IdentityTransform() + ScaledTranslation(
+        ax_x, ax_y, fig.transFigure
+    )
+    # }}}
```

Comment:
I didn't change the transform of the arrows but moved it up here for better reading with the other transforms